Improve layout of lat/lon fields and "Add location" button

mddilley opened the pull request:
Closes #51

mddilley commented:
@mateoclarke That issue with the lat/lon fields and button shifting after submitting a location was due to a header div inserting into the DOM after submission. After removing that div with jQuery upon render of that form's view, the fields/button remain on the map when adding or deleting locations. 
<img width="758" alt="Screen Shot 2019-07-06 at 4 58 11 PM" src="https://user-images.githubusercontent.com/37249039/60761516-85210c80-a00f-11e9-9444-02c3f94abb15.png">

mateoclarke commented on knack/index.js:
```diff
@@ -531,28 +531,22 @@ $(document).on("knack-view-render.view_2609", function(event, scene) {
-  });
-  $form = $("#view_2607");
+// Overlay Latitude/Longitude fields and button on map
+$(document).on("knack-scene-render.scene_1028", function(event, scene) {
```
Looks like we're using the `view-render` event in addition to the `scene-render`. Does the code for each of these things need to be in separate places?

mddilley replied:
The jQuery in the view-render does need to be separate from the scene-render since that view renders after the submission of the lat/lon fields without a full page reload (the jQuery inside the scene-render handles all of the changes for full page reload). There may be a way to combine these, but it gets tricky with the form submission.

mateoclarke commented on knack/index.css:
```diff
@@ -0,0 +1,43 @@
+  display: none;
+}
+
+/* move and style lat/lon fields and button to overlay React app map */
```
I wonder if it would be nice, for the sake of documentation, to add more about where these classes are used.

mddilley replied:
Agreed - it would be very hard to find these targets after moving on from this issue. I added the name of the page and scene id to the comment. Let me know if any other info would help, and I'll add.

mateoclarke commented:
@mddilley I didn't open this up to preview in Knack, but just from your screenshot, it would nice if the lat & lon fields had the same width.

mddilley commented:
@mateoclarke  I adjusted the width of the lat field and move the lon field over to the left. I also moved the labels above the fields and the button below the field.
![Screen Shot 2019-07-08 at 1 49 28 PM](https://user-images.githubusercontent.com/37249039/60834924-75c4cf00-a187-11e9-9b6c-679c02e776d7.png)
